docs/plugins/advertising.md

```diff
@@ -0,0 +1,165 @@
+# Advertising
+
+The _Advertising_ plugin helps to enable (video) advertisements in an App.
+
+The plugin can be used as a drop-in solution that takes care of the entire Ad playback flow (in combination with the VideoPlayer plugin),
+as well as the integration with different advertisment distributers depending on the specific platform deployment.
+
+However if your App requires a custom Advertisement setup with it's own Ad playback logic and integration with an Ad provider, the Advertising
+plugin can be used to retrieve important configuration and settings related to advertisements.
+
+
+## Usage
+
+Whenever you want to enable ads in your App, import the Advertising plugin from the Lightning SDK
+
+```js
+import { Advertising } from '@lightningjs/sdk'
+```
+
+## Drop-in Advertising solution
+
+The drop-in usecase allows you to setup advertisements for your App with minimal code and configuration.
+
+### Setup
+
+In order to enable ads, both the _VideoPlayer_ and _Advertising_ plugin need to be imported. Next the _Advertising_ plugin
```

Comment:
Are we guaranteeing the the new Ad Router-based Ad Framework will work fully with the Lightning SDK player?